src/datasets.c

```diff
@@ -532,6 +532,46 @@ Dataset *DatasetGet(const char *name, enum DatasetTypes type,
     return NULL;
 }
 
+int DatasetReload(uint32_t de_vers)
+{
+    Dataset *set = sets;
+    // Reload if its a static set only
+    while (set != NULL) {
+        // Check if the set is static or dynamic
+        if ((set->save == NULL || strcmp(set->save, "") == 0) || (set->from_yaml == false)) {
+            SCLogInfo("It is a static set or not defined in YAML, so we can continue...");
+        } else {
+            continue;
+        }
+        SCLogInfo("Soon to be hidden Dataset name: %s", set->name);
+        SCLogInfo("Soon to be hidden Dataset type: %d", set->type);
+        set->hidden = true;
+        set = set->next;
+    }
+    return 1;
```

Comment:
Probably don't need to return anything now?

Reply:
agreed